src/components/tiles/AddressTile2.tsx

```diff
@@ -273,12 +310,16 @@ const getStyles = cacheStyles((theme: Theme) => ({
   buttonText: {
     fontSize: theme.rem(0.75),
     color: theme.textLink
+  },
+  loopbackIcon: {
+    transform: [{ rotate: '90deg' }]
```

Comment:
Let's use the other icon.

Reply:
Tentatively selected the AntDesign wallet icon.